src/content/learn/typescript.md

```diff
@@ -0,0 +1,474 @@
+---
+title: Using TypeScript
+re: https://github.com/reactjs/react.dev/issues/5960
+---
+
+<Intro>
+
+TypeScript is a popular way to add type definitions to JavaScript codebases. Out of the box, TypeScript [supports JSX](/learn/writing-markup-with-jsx) and you can get full React Web support by adding [`@types/react`](https://www.npmjs.com/package/@types/react) and [`@types/react-dom`](https://www.npmjs.com/package/@types/react-dom) to your project.
```

Comment:
Is it worth mentioning that the types are official / maintained by the React team?

Reply:
Is it "classed" as being officially maintained by the React team? /cc @eps1lon ?

I thought it wasn't, but I've been out of the game for a few years. If it is, this is definitely something we should add 👍🏻

Reply:
> Is it "classed" as being officially maintained by the React team? 

[Yes](https://react.dev/community/team#sebastian-silbermann)